lib/compiler-llvm/src/translator/code.rs

```diff
@@ -2228,7 +2225,15 @@ impl<'ctx, 'a> LLVMFunctionCodeGenerator<'ctx, 'a> {
                 }
                 */
 
-                let call_site = self.builder.build_call(func, &params, "");
+                let call_site = self.builder.build_call(
+                    func,
+                    params
+                        .iter()
+                        .map(|v| (*v).into())
```

Comment:
⛳ You can also express this as:

```Rust
params.iter().copied().map(Into::into).collect::<Vec<BasicMetadataValueEnum>().as_slice()
```

if you prefer

Reply:
You  can also make `From` impls that accept `&T` as well so you don't have to worry about it being `T` or `&T`, then you can remove the `copied` call here